website/components/remote-plugin-docs/server.js

```diff
@@ -36,6 +40,14 @@ async function generateStaticProps(
     : // Fetch remote content using GitHub's API
       await fetchGithubFile(remoteFile)
   if (err) throw new Error(err)
+  // Construct the githubFileUrl, used for "Edit this page" link
+  // Note: for custom ".docs-artifacts" directories, the "Edit this page"
+  // link will lead to the artifact file rather than the "docs" source file
+  const githubFileUrl = filePath
+    ? `https://github.com/hashicorp/${product.slug}/blob/${mainBranch}/website/${filePath}`
+    : `https://github.com/${remoteFile.repo}/blob/${
+        remoteFile.branch
+      }/${remoteFile.filePath.replace('\b.docs-artifacts', 'docs')}`
```

Comment:
Realized we need to construct the `Edit this page` GitHub URL a little differently for remote plugin docs, hence the related change in `docs-page`. For now this assumes the default `docs` & `.docs-artifacts` locations in the remote repository. We'll need to add configuration to `docs-remote-plugins.json` if we diverge from that pattern.